src/map/script.c

```diff
@@ -2674,11 +2746,14 @@ static struct script_code *parse_script(const char *src, const char *file, int l
 			disp_error_message("unexpected end of script",p);
 		// Special handling only label
 		tmpp=script->skip_space(script->skip_word(p));
-		if(*tmpp==':' && !(strncmp(p,"default:",8) == 0 && p + 7 == tmpp)) {
+		if(*tmpp==':' && !(strncmp(p,"default:",8) == 0 && p + 7 == tmpp) &&
```

Comment:
Missing spaces and please break the line before the logical operator, not after.